stencil.config.ts

```diff
@@ -24,6 +25,11 @@ export const config: Config = {
     slotChildNodesFix: true,
   },
   outputTargets: [
+    reactOutputTarget({
+      componentCorePackage: '../dist/types',
+      proxiesFile: './react-bindings/components.ts',
```

Comment:
This is the output for the generated _source_ typescript React bindings. We still need to compile them to JS in order for them to be used elsewhere.